aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -450,6 +454,10 @@ Tensor resolve_neg(const Tensor& self) {
 // else returns a new negated tensor with neg bit set to 0
 Tensor resolve_conj(const Tensor& self) {
   if (!self.is_conj()) { return self; }
+  // currently a tensor should never have both conj and neg bit set
+  // the only way to get an imag bit is complex_tensor.conj().imag but there's
+  // no intended designed mechanism to enter the complex world with this imag bit
```

Comment:
Well you could do `torch.view_as_complex(t.conj().imag)`.
But I guess you get saved by the fact that imag has a stride of 2 on the last dim and so cannot be viewed as a complex Tensor.

Reply:
exactly that's correct